common/djangoapps/util/views.py

```diff
@@ -434,6 +440,12 @@ def build_error_response(status_code, field, err_msg):
 
     success = False
     context = get_feedback_form_context(request)
+
+    #Update the tag info with 'enterprise_learner' if the user belongs to an enterprise customer.
+    enterprise_learner_data = enterprise_api.get_enterprise_learner_data(site=request.site, user=request.user)
```

Comment:
I could be mistaken, but my understanding was that creating a ticket within the courseware was how this codepath is triggered, so that implies all users coming through here should be authenticated. I think we need to get info about a few of the users from Derek to see if we can find something there.